docs/lab/workflow/order-requirements.mdx

```diff
@@ -4,16 +4,115 @@ title: "Order Requirements"
+As mentioned [in the introduction](/lab/overview/introduction#features), you don't need to have your own Physician to place orders with Vital. When you [order a test](/lab/overview/quickstart#3-placing-an-order), you can pass an optional `physician` argument. If provided, the order will make use of your own physician. If not provided, your order will go through with a physician from our own network. For more information, do not hesitate to head over [our testing page](https://tryvital.io/labs) and book an introductory call with us.
+
+### Health insurance (Beta)
+
+When you [order a test](/lab/overview/quickstart#3-placing-an-order), you can pass an optional `health_insurance` argument to signal that this order should use Health Insurance as the billing method. 
+Depending on the lab parter, a private `physician` may be needed for the order. 
+Currently, we support Health Insurance orders for two of our partner labs, described in the following subsections:
+
+#### Spiriplex
+
+For placing insurance orders with Spiriplex, you will need some pre-requisites:
+
+- A physician capable of signing a requisition form. We currently don't support insurance orders using our own private physician network.
+- A clear, legible image of the front and back of the Patient insurance card
+- A clear, legible image of the Patient signature
+- Information about the patient reported symptoms, and physician assessment and treatment plan
+
+You can provide this information in our API, using the `physician` and `health_insurance` arguments for the [POST v3/order/](/api-reference/lab-testing/create-order) REST endpoint.
+
+```json Health Insurance
+{
+"health_insurance":{
+      "front_image": {
+         "content": "base64-encoded insurance front image binary content",
+         "content_type": "image/png | image/jpg | image/jpeg"
+      },
+      "back_image": {
+         "content": "base64-encoded insurance back image binary content",
+         "content_type": "image/png"
+      },
+      "patient_signature_image": {
+         "content": "base64-encoded patient signature image binary content",
+         "content_type": "image/png"
+      },
+      "subjective": "Description of what is the patient symptoms and attempted treatments",
+      "assessment_plan": "Description of what is the physician assessment and treament or testing plan"
+    }
+}
+```
+
+```json Physician
+{
+"physician": {
+    "first_name": "John",
+    "last_name": "Doe",
+    "email": "john@doe.com",
+    "npi": "123456789",
+    "licensed_states": ["CA", "NY"],
+    "signature_image": {
+         "content": "base64-encoded physician signature image binary content",
+         "content_type": "image/png"
+      }
+  }
+}
+```
+
+After you place an order with all the required information, a requisition form will be generated and sent to Spiriplex
+with the necessary information so your customer’s Health Insurance can be billed by Spiriplex.
```

Comment:
> After you place an order with all the required information, a requisition form will be generated and sent to Spiriplex
with the necessary information so your customer’s Health Insurance can be billed by Spiriplex.

👇 

> After you place an order with all the required information, a requisition form is generated and sent to Spiriplex
with all the necessary information. Your customer’s Health Insurance can then be billed by Spiriplex.